File: CSharp/LeetCode/CountAsterisks.cs
// 2315. Count Asterisks
public class Solution {
    public int CountAsterisks(string s) {
        int count = 0;
        string[] words = s.Split("|");

        for (int i = 0; i < words.Length; i++)
            if (i % 2 == 0)
                count += words[i].Count(x => x == '*');

        return count;

    }
}

File: Python/LeetCode/countAsterisks.py
# 2315. Count Asterisks
class Solution:
    def countAsterisks(self, s: str) -> int:
        count = 0
        words = s.split("|")

        for i in range(len(words)):
            if i % 2 == 0:
                count += words[i].count("*")
        
        return count